test/integration/conftest.py

```diff
@@ -0,0 +1,60 @@
+import os
+import subprocess
+
+import tempfile
+import uuid
+
+import pytest
+
+
+@pytest.fixture
+def build_dir_and_ee_yml():
```

Comment:
it's not out of the question that this could be shared with the existing unit tests. As we get into the think of things, that's what I would prefer.

Reply:
cool can move it up a level if you want to share this fixture